pkg/minishift/openshift/service.go

```diff
@@ -86,12 +103,20 @@ func GetServiceURLs(serviceListNamespace string) ([]ServiceURL, error) {
 
 	// iterate over namespaces, get command output, format route in ServiceURL
 	for _, namespace := range namespaces {
-		outputData, err := getServiceURLsOutput(namespace)
+		serviceNodePort, err := getService(namespace, ip)
+		if err != nil {
+			fmt.Println("Here is errror")
+			return serviceURLs, err
+		}
+		outputData, err := getRouteURLsOutput(namespace)
```

Comment:
_routes_ instead of _ outputData_ and _getRoutes_ instead of _getRouteURLsOutput_ ?